bigquery/src/test/java/com/google/cloud/hadoop/io/bigquery/RegionalIntegrationTest.java

```diff
@@ -180,7 +181,10 @@ public void setup() throws Exception {
         new Storage.Builder(
                 HTTP_TRANSPORT,
                 JSON_FACTORY,
-                new RetryHttpInitializer(GoogleCloudStorageTestHelper.getCredential(), BQC_ID))
+                new RetryHttpInitializer(
+                    /* delegate= */ null,
```

Comment:
nit : we could use constructor without delegate

Reply:
I removed, but plan to clean this up in follow up CL - maybe there a way to avoid delegate at all, if not I will add a builder.